android-agent/src/main/java/io/opentelemetry/android/internal/services/network/CurrentNetworkProvider.java

```diff
@@ -107,11 +102,11 @@ private static NetworkRequest createNetworkMonitoringRequest() {
                 .build();
     }
 
-    CurrentNetwork getCurrentNetwork() {
+    public CurrentNetwork getCurrentNetwork() {
         return currentNetwork;
     }
 
-    void addNetworkChangeListener(NetworkChangeListener listener) {
+    public void addNetworkChangeListener(NetworkChangeListener listener) {
```

Comment:
Is this mutability exposed to facilitate testing? Should we make it package then?

Reply:
Sorry I'm not seeing it. Can you point to where? 🙏🏻

Reply:
Ah, gotcha. Yeah it seems to be part of the code not shown by GH diff. It's in [NetworkApplicationListener](https://github.com/open-telemetry/opentelemetry-android/pull/408/files#diff-811ab73f473dc6d7c99372a0cce7f2140b7afef8e5665e4539e9ae00ba66c083) line 30 (you'd have to expand the file to see it).

Reply:
ok, thanks.